test/unittest/unittest_nntrainer_tensor.cpp

```diff
@@ -3393,7 +3393,7 @@ TEST(nntrainer_Tensor, split_01_p) {
                              110, 111, 112, 113, 114, 115, 116, 117, 118, 119};
       answer.emplace_back(ml::train::TensorDim{1, 2, 4, 5}, answer_data);
     }
-    EXPECT_EQ(t.split(3, 0), answer);
+    EXPECT_EQ(t.split({3}, 0), answer);
```

Comment:
you don't need to change these unit test once you provide ```td::vector<Tensor> Tensor::split(unsigned int num_size, int axis)```

Reply:
Reverted